### PR TITLE
[9.0] Added Pilot tests to integration tests

### PR DESCRIPTION
**Pull request by fstagni**
This PR adds a new "pilot" integration test. It basically runs the pilot into yet another container, and runs jobs. In a way, this completes the "full grid" test.

BEGINRELEASENOTES

*test
NEW: added pilot workflow tests to integration_tests

ENDRELEASENOTES

### CODE REVIEW

File: tests/Integration/WorkloadManagementSystem/Test_TaskQueueDB.py [hunk 1]
```diff
@@ -65,13 +65,6 @@ def test_chainWithParameter():
     assert result["Value"] is False
     result = tqDB.retrieveTaskQueues()
     assert result["OK"]
-    assert list(result["Value"].values())[0] == {
-        "Owner": "userName",
-        "Jobs": 2,
-        "OwnerGroup": "myGroup",
-        "CPUTime": 86400,
-        "Priority": 1.0,
-    }
```
Comment (aldbr) [inline tests/Integration/WorkloadManagementSystem/Test_TaskQueueDB.py:74]: Why did you remove this `assert`?

**Review by aldbr — APPROVED** [review]

File: tests/Integration/WorkloadManagementSystem/Test_TaskQueueDB.py (hunk 1 above)
Comment (fstagni) [reply tests/Integration/WorkloadManagementSystem/Test_TaskQueueDB.py:74]: Because, since with this PR some jobs can be submitted and some pilots run, we can't with certainty know we can expect to see in the DB.
